Homework/Hang_Nguyen/lesson_5/summary-about-ACID.md

```diff
@@ -0,0 +1,26 @@
+<h3>ACID stands for:</h3>
```

Comment:
this is a good start. I suggest add more detail, recommend watching this video https://youtu.be/5ZjhNTM8XU8 if you don't like reading text book DDIA